### PR TITLE
Add check for expired meetings

### PR DESCRIPTION
**Pull request by tonishka**
1. Meeting card displays user-friendly date time
2. Meeting list does not display expired meetings (it is not deleted in the backend however)

### CODE REVIEW

File: src/main/java/seedu/address/model/meeting/MeetingTime.java [hunk 1]
```diff
@@ -39,6 +39,16 @@ public static boolean isValidMeetingTime(String test) {
...
     }
 
+    /**
+     * Returns true if a given meeting has expired.
+     */
+    public boolean isExpiredMeetingTime() {
+        if (this.dateTime.isBefore(LocalDateTime.now())) {
+            return true;
+        }
+        return false;
```
Comment (ckcherry23) [inline src/main/java/seedu/address/model/meeting/MeetingTime.java:49]:
```suggestion
        return this.dateTime.isBefore(LocalDateTime.now());
```

**Review by ckcherry23 — APPROVED** [review]

File: src/main/java/seedu/address/model/meeting/MeetingTime.java [hunk 2]
```diff
@@ -39,6 +39,13 @@ public static boolean isValidMeetingTime(String test) {
         }
     }
 
+    /**
+     * Returns true if a given meeting has expired.
+     */
+    public boolean isExpiredMeetingTime() {
+        return this.dateTime.isBefore(LocalDateTime.now());
```
Comment (NicsunXnus) [inline src/main/java/seedu/address/model/meeting/MeetingTime.java:46]: I am confused, shouldn't the meeting time expire when the datetime is after the current time? Here the method runs true if the date time is before the current time.

Comment (ckcherry23) [reply src/main/java/seedu/address/model/meeting/MeetingTime.java:46]: If the meeting is before now, it has expired right? If it after the current time, it should be in the upcoming meetings list.

Comment (NicsunXnus) [reply src/main/java/seedu/address/model/meeting/MeetingTime.java:46]: oh yeah, ok that clears things up

**Pull request merged**